pytest_factoryboy/__init__.py

```diff
@@ -1,5 +1,6 @@
 """pytest-factoryboy public API."""
-from .fixture import register, LazyFixture
+from .fixture import LazyFixture
+from .fixture import register
```

Comment:
why the previous approach was wrong?

Reply:
This is using reorder_python_imports, the same as pytest

Reply:
please revert this. Let's have a PR with one objective, if we want to introduce a new tool to this project it should be a separate PR where we can discuss that.

Reply:
@youtux I've reverted the change to use reorder-python-imports, and will make it an a followup